src/test/test_data.ts

```diff
@@ -0,0 +1,5 @@
+module. exports = {
```

Comment:
What is the purpose of this separate file here? Can these few settings here not be included in the file config.ts just like TEST_URL?

Reply:
Removed test-data file and moved data to Config file